watertap/property_models/anaerobic_digestion/adm1_properties_vapor.py

```diff
@@ -0,0 +1,466 @@
+                        - False - state variables are unfixed after
+                                 initialization by calling the
+                                 release_state method
+
+        Returns:
+            If hold_states is True, returns a dict containing flags for
+            which states were fixed during initialization.
+        """
+        init_log = idaeslog.getInitLogger(blk.name, outlvl, tag="properties")
+
+        if state_vars_fixed is False:
+            # Fix state variables if not already fixed
+            flags = fix_state_vars(blk, state_args)
+
+        else:
+            # Check when the state vars are fixed already result in dof 0
+            for k in blk.keys():
+                if degrees_of_freedom(blk[k]) != 0:
+                    raise Exception(
+                        "State vars fixed but degrees of freedom "
+                        "for state block is not zero during "
+                        "initialization."
+                    )
+
+        if state_vars_fixed is False:
+            if hold_state is True:
+                return flags
+            else:
+                blk.release_state(flags)
+
+        init_log.info("Initialization Complete.")
+
+    def release_state(blk, flags, outlvl=idaeslog.NOTSET):
+        """
+        Method to release state variables fixed during initialization.
+
+        Keyword Arguments:
+            flags : dict containing information of which state variables
+                    were fixed during initialization, and should now be
+                    unfixed. This dict is returned by initialize if
+                    hold_state=True.
+            outlvl : sets output level of logging
+        """
+        init_log = idaeslog.getInitLogger(blk.name, outlvl, tag="properties")
+
+        if flags is None:
+            return
+        # Unfix state variables
+        revert_state_vars(blk, flags)
+        init_log.info("State Released.")
+
+
+@declare_process_block_class("ADM1_vaporStateBlock", block_class=_ADM1_vaporStateBlock)
+class ADM1_vaporStateBlockData(StateBlockData):
+    """
+    StateBlock for calculating thermophysical properties associated with the ADM1
+    reaction system.
+    """
+
+    def build(self):
+        """
+        Callable method for Block construction
+        """
+        super().build()
+
+        # Create state variables
+        self.flow_vol = pyo.Var(
+            initialize=1,
+            domain=pyo.NonNegativeReals,
+            doc="Total volumetric flowrate",
+            units=pyo.units.m**3 / pyo.units.s,
+        )
+        self.pressure = pyo.Var(
+            domain=pyo.NonNegativeReals,
+            initialize=101325.0,
+            bounds=(1e3, 1e6),
+            doc="Pressure",
+            units=pyo.units.Pa,
+        )
+        self.temperature = pyo.Var(
+            domain=pyo.NonNegativeReals,
+            initialize=298.15,
+            bounds=(298.15, 323.15),
```

Comment:
Revisit temp range? I'd expect a higher upper bound but don't know how much it matters in the context of our current model and usage. #909